tests/test.js

```diff
@@ -154,3 +176,49 @@ test('test getTraversableCells', t => {
 
   t.same(helpers.getTraversableCells(cpt, smallMap, me), expected);
 });
+
+test('test init2dArray', t => {
+  t.plan(2);
+
+  let width = 5;
+  let height = 3;
+  let defaultVal = 1;
+  let expected = [
+    [1, 1, 1],
+    [1, 1, 1],
+    [1, 1, 1],
+    [1, 1, 1],
+    [1, 1, 1],
+  ];
+  t.same(helpers.init2dArray(width, height, defaultVal), expected);
+
+  width = 3;
+  height = 3;
+  defaultVal = 55;
+  expected = [
+    [55, 55, 55],
+    [55, 55, 55],
+    [55, 55, 55],
+  ];
+  t.same(helpers.init2dArray(width, height, defaultVal), expected);
+});
+
+test('test multiplyCorrespondingElementsAndSum', t => {
+  t.plan(1);
+
+  const m1 = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+  ];
+
+  const m2 = [
+    [9, 8, 7],
+    [6, 5, 4],
+    [3, 2, 1],
+  ];
+
+  const expected = 165;
+
+  t.same(helpers.multiplyCorrespondingElementsAndSum(m1, m2), expected);
```

Comment:
`t.same` is for deep-equals comparison (ie, checking if arrays are equal). for comparing primatives you should use `t.is`. Doesn't actually change anything, but is convention. See this for reference:

https://github.com/substack/tape